dot/core/service.go

```diff
@@ -252,11 +250,12 @@ func (s *Service) handleBlock(block *types.Block, state *rtstorage.TrieState) er
 }
 
 func (s *Service) handleCodeSubstitution(hash common.Hash, state *rtstorage.TrieState) error {
-	return s.handleCodeSubstituionWithWasmerInstance(hash, state)
+	return s.handleCodeSubstitutionWithWasmerInstance(hash, state, wasmer.NewInstance)
 }
-func (s *Service) handleCodeSubstituionWithWasmerInstance(
+func (s *Service) handleCodeSubstitutionWithWasmerInstance(
```

Comment:
can you change it back to `_handleCodeSubstitution`.  It's understood that you're doing this to make the `handleCodeSubstitution` more testable.  The new name implies it's doing something different and is called by other receiver methods.

Reply:
will do!